Xamarin.Forms.Xaml.UnitTests/Xamarin.Forms.Xaml.UnitTests.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NUnit" Version="2.6.4" />
-    <PackageReference Include="Microsoft.Build.Locator" Version="1.0.31" />
+    <PackageReference Include="Microsoft.Build.Locator" Version="1.2.6" />
```

Comment:
?

Reply:
This is needed to get the 4.1.0 unit tests to pass and Rui already applied this into 4.1.0
https://github.com/xamarin/Xamarin.Forms/pull/7783/files